### PR TITLE
Added new keyword : BUG and also added python support

### PR DESCRIPTION
**Pull request by hemantjadon**
Please Verify and merge

### CODE REVIEW

**Comment by hemantjadon** [comment]
Please Verify and give feedback .. :)

**Comment by hemantjadon** [comment]
Status ?? Actually this is my first open source contribution .. so just a little anxious :P

**Comment by 50Wliu** [comment]
Don't worry, someone will get to reviewing this eventually.  I'm a bit busy at the moment but I'll try to get to this later this week.

**Comment by 50Wliu** [comment]
Ok, there's a few issues with this PR:
First of all, the top-level selector for Python is `.source.python`, not `.text.python`.  If you tested this PR out it shouldn't have worked on Python files.

Second, there's no need to create another section when you could have just added `.source.python` to the list on line 1.

Other than that the `BUG` addition looks good to me.

**Comment by hemantjadon** [comment]
Done the mentioned changes. used selector as `.source.python` instead of `.text.python`
also combined it with the common list. Thanks :)

File: snippets/todo.cson [hunk 1]
```diff
@@ -1,4 +1,4 @@
-'.source.ruby, .source.puppet, .source.coffee':
+'.source.ruby, .source.puppet, .source.coffee .source.python':
```
Comment (50Wliu) [inline snippets/todo.cson]: This needs a comma before it just like the others.

**Comment by hemantjadon** [comment]
Thank you .. :) 
Actually I am not able to test the code, as i don't know how to use this `.cson` file with atom instead of its original one.
If you can point out how to do that it will be great

**Comment by 50Wliu** [comment]
1. Clone this package (or your fork) using `git`
2. Navigate to where you cloned it and `apm link` (or `apm link --dev`)
3. Open Atom like usual (or `atom --dev` if you used the latter option)

**Pull request merged**